.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,23 @@
+name: PHP_CodeSniffer
+
+on: pull_request
+
+jobs:
+  phpcs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+          coverage: none
+          tools: composer
+
+      - name: Install dependencies
+        run: composer install --prefer-dist --no-suggest --no-progress
+
+      - name: Detect coding standard violations
+        run: vendor/bin/phpcs
```

Comment:
@shubhw12 We have done same changes in this PR as well, can you please check? - https://github.com/brainstormforce/custom-fonts/pull/40/files - Can we close this then?